src/redux/Graphql/saga.js

```diff
@@ -192,6 +349,10 @@ export function* getPendingTransactionsHandler() {
       yield put({
         type: actions.GET_PENDING_TRANSACTIONS_RETURN,
         value: [],
+        error: {
```

Comment:
these background graphql calls dont need error objects since we fail silently, remember?